YvesIraguha/yves-iraguha-portfolio
# Home page

#### What does this PR do?
- set up react-router 
- build home page
#### Description of Task to be completed?
- complete the home page
#### How should this be manually tested?
- Clone the repo
- checkout `home-page` branch
- Install all dependencies by `yarn` and start the server by `yarn run start`
- Open localhost on port `3000` you should see the home page
#### Any background context you want to provide?
- This home page, come to replace an already existing home page

#### Screenshots (if appropriate)
![Screen Shot 2019-10-07 at 13 34 53](https://user-images.githubusercontent.com/34691070/66308370-4600a100-e907-11e9-8dfb-207820923017.png)
![Screen Shot 2019-10-07 at 13 36 29](https://user-images.githubusercontent.com/34691070/66308453-819b6b00-e907-11e9-8fdf-4acd056102ce.png)

#### Questions:
